tests/tests/dos.rs

```diff
@@ -1,11 +1,18 @@
 #![allow(warnings)]
 
+use std::time::{
+    Duration,
+    Instant,
+};
+
+use ethers::abi::Tokenizable;
```

Comment:
curious where this is being used

Reply:
Same

Reply:
Fixed in https://github.com/FuelLabs/fuel-core/pull/2309/commits/83103c0834321de0c28d584ff5da873159a4c8a9